lib/generators/active_admin/install/install_generator.rb

```diff
@@ -38,6 +38,15 @@ def setup_routes
       end
 
       def create_assets
+        js_file = File.join(destination_root, "app", "assets", "javascript", "application.js")
+        css_file = File.join(destination_root, "app", "assets", "stylesheets", "application.css")
+        if File.foreach(css_file).grep(/\*= require_tree \./).any? || File.foreach(js_file).grep(/\/\/= require_tree \./).any?
+          say_status(
+            :warning, "The code #{set_color("require_tree .", :red)} in either application.css or application.js files
+              makes Active Admin styles or scripts to be included in other parts of your app.
+              If that is not desired you can stub active_admin at the end of those files.")
```

Comment:
This looks good to me! With the stubbing I guess you're referring to https://github.com/activeadmin/activeadmin/issues/2514#issuecomment-462473973. If that's the case, I think we could be more explicit recommending the exact solution? Also, an alternative solution would be to completely remove the `require_tree .` in favour of explicit requires, correct?